### PR TITLE
We're hiring!

### PR DESCRIPTION
**Pull request by skade**
@badboy can I have some lightweight review on this, especially the german translation?

### CODE REVIEW

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+layout: post
+title: "Wir stellen ein für Buchhaltung und Management!"
+permalink: wir-stellen-ein-fuer-buchhaltung-und-management
+---
+
+English version [here](/we-re-hiring-for-bookkeeping-and-event-management).
+
+## Wir stellen ein!
+
+[RustFest](https://rustfest.eu) ist die europäische Rust-Konferenz. Wir sehen uns als Schaukasten für alles was in der Rust Community so passiert. Jedes RustFest findet and einem anderen Ort statt, wechselnd alle 6 Monate. Das RustFest findet im zum 5ten mal statt.
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- and einem anderen
+ an einem anderen
```

Comment (badboy) [reply _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- Das RustFest findet im zum 5ten mal statt.
+ RustFest findet im Herbst 2018 zum fünften mal statt.
```

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+Wir haben uns mit [Ruby Berlin e.V.](https://rubyberlin.org) and [OpenTechSchool e.V.](https://opentechschool.org) zusammengetan, um eine Stelle für Buchhaltung und Management zu schaffen. Die Buchhaltung ist momentan der größte Teil.
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- and
+ und
```

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+Alle diese Projekte sind stolz darauf, eine inklusive Umgebung zu schaffen und aktiv daran zu arbeiten, diese ständig zu verbessern.
+
+Bisher sind all diese Projekte auf rein freiwilliger Basis und viele Beteiligte übernehmen mehrere Rollen. Wir stellen fest, dass eine ansprechbare Buchhaltung und eine ansprechbare Geschäftsstelle für einfache Nachfragen viel helfen würden.
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch 
- auf rein freiwilliger Basis
+ auf rein freiwilliger Basis organisiert
```

Comment (badboy) [reply _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- Wir stellen fest,
+ Wir haben festgestellt,
```

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+Der Job besteht aus:
+* Buchhaltung RustFest (2 Mal pro Jahr)
+  - Bearbeitung von Sponsor-Rechnungen
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- Sponsor-Rechnungen
+ Sponsorrechnungen
```

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+  - Bearbeitung von Rückerstattungen 
+  - Bearbeitung von Rückerstattungen von Speaker-Kosten und Diversity-Tickets
+  - Jährliche Steuererklärung (zusammen mit Steuerberater)
+* Buchhaltung Ruby Berlin e.V.
+  - Bearbeitung Sponsor-Rechnungen zusammen mit Code Curious
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- Sponsor-Rechnungen
+ Sponsorrechnungen
```

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+  - Bearbeitung allgemeiner Vereins-Rechungen
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- Vereins-Rechnungen
+ Vereinsrechnungen
```

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+  - Unterstützung des Vorstands
+  - Jährliche Steuererklärung (zusammen mit Steuerberater)
+* Bookkeeping OpenTechSchool e.V.
+  - Bearbeitung Sponsor-Rechnungen
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- Sponsor-Rechnungen
+ Sponsorrechnungen
```

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+  - Bearbeitung allgemeiner Vereins-Rechungen
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- Vereins-Rechnungen
+ Vereinsrechnungen
```

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+  - Unterstützung des Vorstands
+  - Jährliche Steuererklärung
+* Geschäftsstelle für alle drei
+  - Antworten auf Fragen allgemeiner Art
+  - Kontakt halten mit Sponsoren
+
+Beachte dass sowohl Ruby Berlin als auch OpenTechSchool nicht viele Rechnungen bekommen, die meisten Wochen vergehen ohne anfallende Arbeiten. Die Arbeiten müssen nicht aus dem Büro geschehen, aber wir können auf Wunsch einen Arbeitsplatz stellen. Alle notwendigen Arbeitsmittel (computer, etc.) können gestellt werden.
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- Beachte dass
+ Beachte, dass
```

Comment (badboy) [reply _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- computer, etc.
+ Computer, etc.
```

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+
+Alles zusammen erwarten wir ca. 30 Stunden *monatlich* für alle drei.
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- für alle drei
+ für alle drei Organisationen
```

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+
+## Was wir anbieten
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- Was wir anbieten
+ Was wir bieten
```

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+
+Wir bieten eine flexible Arbeitsumgebung in freundlichen Organisationen, die sich der Ausbildung und der Vielfältigkeit im Techonologie-Sektor verschrieben haben.
+
+Im Detail:
+* Eine Heimarbeitsstelle
+* 25EUR Lohn pro Stunde (verhandelbar)
+* Flexible Arbeitszeiten, mit kaum harten Zeitanforderungen ausserhalb der Konferenzwochen
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- ausserhalb
+ außerhalb
```

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 13]
```diff
@@ -0,0 +1,92 @@
...
+* Eine freundliche Arbeitsumgebung
+* Unterstützung und Ausbildung, solltest du dir Dinge auf Arbeit aneignen müssen
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- solltest du dir Dinge auf Arbeit aneignen müssen
+ solltest du dir Dinge bei der Arbeit aneignen müssen
```

I'm not happy with the sentence here. hm....

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 14]
```diff
@@ -0,0 +1,92 @@
...
+
+Dieser Job ist eine gute Gelegenheit für Menschen, die eine flexible Stelle mit Heimarbeit und flexiblen Arbeitszeiten brauchen.
+
+Die Anstellung wird durch die [asquera GmbH](https://asquera.de) geschehen.
+
+## Wonach wir suchen
+
+Du bist daran interessiert, Aktiven bei ihrer gemeinnützigen Arbeit unter die Arme zu greifen.
+
+* Du erledigst Büroarbeit auf organisierte Art und weise
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- auf organisierte Art und weise
+ auf organisierte Art und Weise
```

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 15]
```diff
@@ -0,0 +1,92 @@
...
+* Du kannst einfache Buchhaltung
+  - Die Leitung der Organisationen wird dir dabei helfen
+  - Wir haben Steuerfachkräfte für die schwierigen Fragen
+* Du kannst email auf Deutsch und Englisch beantworten
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- email
+ Email
```

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 16]
```diff
@@ -0,0 +1,92 @@
...
+  - Beides müssen nicht deine Muttersprache sein, aber du solltest halbwegs sicher sein
+* Du bist selbstorganisiert
+  - Du wirst mit Freiwilligen arbeiten und bist die einzige Person, die bezahlt wird
+* Du kannst dich mit um unseren Jahresrythmus organisieren
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- dich mit um unseren Jahresrythmus organisieren
+ dich mit mit unserem Jahresrythmus organisieren
```

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 17]
```diff
@@ -0,0 +1,92 @@
...
+  - RustFest findet 2 Mal im Jahr statt, eine Ansprechbarkeit in diesen 2 Wochen ist wichtig
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]:
```patch
- eine Ansprechbarkeit in diesen 2 Wochen ist wichtig
+ eine Ansprechbarkeit in den 2 Wochen der Konferenzen ist wichtig

File: _posts/2018-07-24-wir-stellen-ein.md [hunk 18]
```diff
@@ -0,0 +1,92 @@
...
+* Du bist flexibel
+  - Wir würden uns freuen, wenn du im Rahmen deiner Zeit weitere Aufgaben übernehmen kannst
+* Du verstehst und unterstützt unsere [Ziele zu Diversität und Empowerment](http://www.opentechschool.org/about.html)
+  - Auch hier freuen wir uns über Mithilfe, aber das steht dir ganz frei
+
+Beachte, dass Reisen nicht notwendig ist, aber wenn du unsere Veranstaltungen besuchen möchtest, übernehmen wir selbstverständlich die entstehenden Kosten.
+
+Immernoch interessiert? [Schreibe uns!](mailto:florian@rustfest.eu).
```
Comment (badboy) [inline _posts/2018-07-24-wir-stellen-ein.md]: Remove the "Immernoch"

**Review by badboy — CHANGES_REQUESTED** [review]
Can't believe how difficult it is to read a job posting in my first language, after I read the english one already. :D 

English one seems good. I left a couple of suggestions on the German one.

**Comment by skade** [comment]
@badboy Fixed all your issues.

**Review by badboy — APPROVED** [review]

**Pull request merged**